README.md

```diff
@@ -18,7 +18,7 @@ TimeZones.jl
 * ZonedDateTime-Period arithmetic [similar to that of DateTime](https://docs.julialang.org/en/stable/manual/dates/#TimeType-Period-Arithmetic-1)
 * Local system time zone information as a TimeZone
 * Current system time in any TimeZone
-* Support for reading the [tzfile](http://man7.org/linux/man-pages/man5/tzfile.5.html) format
+* Support for reading the [tzfile](https://man7.org/linux/man-pages/man5/tzfile.5.html) format
```

Comment:
It would be best if you make these kind of changes in a separate PR then they can get reviewed and merged independently

Reply:
Ok. I wanted to make the whole CI thing passing before merging the PR, but I'll leave it for another PR.

Reply:
I'll extracted this as a separate PR: https://github.com/JuliaTime/TimeZones.jl/pull/273

Reply:
Ok, reverted to http in this PR